.pre-commit-config.yaml

```diff
@@ -47,6 +47,14 @@ repos:
         name: gen-docs
         entry: python hooks/gen_docs.py
         language: system
-        types: [python]
+        files: > # contains `.gitignore` as an indicator whether the script is ran with `--all-files`
```

Comment:
clever, any side-effects if we purposefully make changes to the gitignore?

Reply:
The warnings wouldn't appear, nothing more.

Reply:
great!